cli/config/config.go

```diff
@@ -139,6 +141,9 @@ func NewJobConfiguration(cfgFilePath string) (Project, error) {
 		c.Env = make(map[string]string)
 	}
 
+	// TODO: should be remove after deprecate capabilities field
```

Comment:
Somewhat of a misnomer comment, since with your changes, the `capabilities` field has been officially deprecated ;)

Reply:
Yes. `capabilities` only exists in config yml files. Let me update this a bit.